specs/permissions-deployment-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "type": "object",
+    "properties": {
+        "permissionDeployments": {
+            "type": "object",
+            "patternProperties": {
+                "[\\w]+\\.[\\w]+[\\.[\\w]+]?": {
+                    "type": "object",
```

Comment:
Shouldn't this be an array of objects?

Reply:
Fixed in https://github.com/microsoftgraph/kibali/pull/42